src/Fmt_ast.ml

```diff
@@ -1420,9 +1425,11 @@ and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext ({ast= exp} as xexp)
           ( wrap_if parens "(" ")"
               (hovbox 0
                  ( hovbox 2
-                     ( hovbox 2
+                     ( ( match c.conf.fun_call with
+                       | `Sparse -> hvbox 2
+                       | `Compact -> hovbox 2 )
```

Comment:
Maybe use `@@` for these applications where the operator is a complex expression